predicts_api.py

```diff
@@ -18,14 +17,15 @@ def predict():
     if request.method == 'POST':
         try:
             json_data = request.get_json()
+            print(json_data)
             if json_data is None:
                 return jsonify("Expected json")
             else:
-                df = pd.DataFrame({0: json_data}).transpose()
+                data = [list(d.values()) for d in json_data]
```

Comment:
Is it safe to just convert dict to an array? What order of fields do you expect? What can you really receive?

Reply:
`d.values()` keeps the same order as it was in dict.